package.json

```
@@ -1032,6 +1037,10 @@
           "when": "view == confluent-resources && viewItem =~ /.*-kafka-cluster.*/",
           "group": "2_copy@3"
         },
+        {
+          "command": "confluent.resources.scaffold",
+          "when": "viewItem =~ /.*-kafka-cluster.*/ || viewItem =~ /.*-topic.*/"
```

Just to be on the safe side so we don't accidentally match another extension's tree items that may include `-kafka-cluster` or `-topic`
```suggestion
          "when": "view in confluent.viewsWithResources && (viewItem =~ /.*-kafka-cluster.*/ || viewItem =~ /.*-topic.*/)"
```